From: Bhushan Shah <bhush94@gmail.com>
Date: Tue, 16 Jun 2015 12:05:00 +0000
Subject: Remove X-KDE-Library from the trash library
X-Git-Url: http://quickgit.kde.org/?p=plasma-desktop.git&a=commitdiff&h=f0d4cce819b3ce683a11b49a93ba81bafef7a4dd
---
Remove X-KDE-Library from the trash library

trash applet is not using the c++ applet part, so remove it

BUG: 349207
---


--- a/applets/trash/package/metadata.desktop
+++ b/applets/trash/package/metadata.desktop
@@ -82,7 +82,6 @@
 X-Plasma-API=declarativeappletscript
 X-Plasma-MainScript=ui/main.qml
 
-X-KDE-Library=plasma_applet_trash
 X-KDE-PluginInfo-Author=Heena
 X-KDE-PluginInfo-Email=heena393@gmail.com
 X-KDE-PluginInfo-Name=org.kde.plasma.trash

